clang/lib/Tooling/Refactoring/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ add_clang_library(clangToolingRefactoring
   Rename/USRFinder.cpp
   Rename/USRFindingAction.cpp
   Rename/USRLocFinder.cpp
+  SymbolName.cpp
```

Comment:
Should `SymbolName.cpp` live in `Rename` to match its header?

Reply:
Goode suggestions 👍🏽